src/main/java/homework_7/kitten_to_cat/Cat.java

```diff
@@ -0,0 +1,56 @@
+package homework_7.kitten_to_cat;
+
+import java.security.SecureRandom;
+import java.util.Random;
+
+public class Cat {
+
+    private final int age;
+    private final String name;
```

Comment:
Opt: you could provide such fields and such lambda, that it'll make type conversion